packages/table/test/tableTests.tsx

```diff
@@ -783,4 +857,19 @@ describe("<Table>", () => {
     function renderCell() {
         return <Cell>gg</Cell>;
     }
+
+    function updateLocatorBodyElement(table: ReactWrapper<any, {}>,
+                                      left: number,
+                                      top: number,
+                                      width: number,
+                                      height: number) {
```

Comment:
hmm i'd be inclined to name these args for the HTML properties they represent. 
left => scrollLeft, height => clientHeight, etc.